source/mainnet/net/voting/voting.rst

```diff
@@ -118,6 +120,20 @@ These older wallets do not have the capability to connect to dApps. But you do h
 
    #. It is possible to check from the Voting DApp that the delegation worked. The window for delegating votes is the same as voting with the DApp.
 
+.. dropdown:: Concordium Client
+
+    Users need to create a new account in |bw| or |cryptox| and delegate their vote from their Concordium Client wallet account(s) to the new account, then vote from the new account. You can create the new account in |bw| or |cryptox| at any time before the election.
+
+    **To delegate your vote:**
+
+    .. code-block:: console
+
+        $concordium-client transaction send --amount AMOUNT --receiver A --sender B --memo delegatevote2024
+
+    #. Enter the command above in the Concordium Client, where AMOUNT is the number of CCD to be sent (1 micro-CCD is enough), A is the name/address of the account to which the vote is delegated, and B is an optional sender name/address, which is needed only if multiple accounts have been imported in the Concordium Client. Note the memo *delegatevote2024* for the delegation to be valid.
```

Comment:
```suggestion
    #. Enter the command above in the Concordium Client, where AMOUNT is the number of CCD to be sent (1 micro-CCD is enough), A is the name/address of the account to which the vote is delegated, and B is sender name/address. Note the memo *delegatevote2024* for the delegation to be valid.
```

In my opinion, it would be better if they always try to add the name or address to be sure it is clear what is going to happen.